pipelines/tekton/pipeplines_provider.go

```diff
@@ -183,6 +199,61 @@ func (pp *PipelinesProvider) Run(ctx context.Context, f fn.Function) error {
 	return nil
 }
 
+// Creates tar stream with the function sources as they were in "./source" directory.
+func sourcesAsTarStream(f fn.Function) *io.PipeReader {
```

Comment:
Shouldn't we respect `.gitignore` file in here?

Reply:
💯